staging/src/k8s.io/client-go/listers/rbac/v1/clusterrole.go

```diff
@@ -63,3 +67,52 @@ func (s *clusterRoleLister) Get(name string) (*v1.ClusterRole, error) {
 	}
 	return obj.(*v1.ClusterRole), nil
 }
+
+// ClusterRoles returns an object that can list and get ClusterRoles.
+func (s *clusterRoleLister) ClusterRoles() ClusterRoleTenantLister {
+	return clusterRoleTenantLister{indexer: s.indexer, tenant: "default"}
```

Comment:
same question here about "default" tenant

Reply:
That is the backward-compatible version, where the tenant value is set to "default".

There is new version, ClusterRolesWithMultiTenancy(), which allows the "tenant" to take any value.